shared/store/wallet.ts

```diff
@@ -211,9 +213,27 @@ export const wallet = createSlice({
       fetchPoolPairs.fulfilled,
       (state, action: PayloadAction<DexItem[]>) => {
         state.hasFetchedPoolpairData = true;
-        state.poolpairs = action.payload.filter(
-          ({ data }) => !data.symbol.includes("/v1")
-        ); // Filter out v1 pairs due to stock split
+
+        // Get pre stocksplit stocks
+        const stockSplitPairs = action.payload.filter(({ data }) => {
+          return data.symbol.includes("/v1");
+        });
+
+        // Filter out v1 pairs due to stocksplit, excluding post stocksplit (without "/v1" indicator)
+        state.poolpairs = action.payload
+          .filter(({ data }) => !data.symbol.includes("/v1"))
+          .map((item) => {
+            return {
+              ...item,
+              // poolpairs with added hasStockSplit boolean indicator for post stocksplit
+              hasStockSplit: stockSplitPairs.some((pair) => {
+                return (
+                  pair.data.symbol === `${item.data.symbol}/v1` ||
+                  item.data.symbol.includes("/v1")
```

Comment:
```suggestion
                  pair.data.symbol === `${item.data.symbol}/v1`
```
condition not needed right cos all the string with `v1` is filtered out on line 224?

Reply:
If item.data.symbol.includes("/v1") is not included in the condition then the hasStockSplit will be set to false instead of true